07-the-model-workflow.Rmd

```diff
@@ -208,10 +210,9 @@ The solution in `r pkg(workflows)` is an optional supplementary model formula th
 
 ```{r workflows-strata}
 library(survival)
+library(censored)
```

Comment:
Do you want to explicitly load both? Or just censored? We don't typically show loading other engine packages.

Reply:
FYI censored depends on survival so if this is to get access to `Surv()` without namespacing it, this is not needed when loading censored